src/ui/public/agg_types/__tests__/metrics/parent_pipeline.js

```diff
@@ -1,17 +1,17 @@
 import expect from 'expect.js';
 import ngMock from 'ng_mock';
-import { AggTypesMetricsDerivativeProvider } from 'ui/agg_types/metrics/derivative';
-import { AggTypesMetricsCumulativeSumProvider } from 'ui/agg_types/metrics/cumulative_sum';
-import { AggTypesMetricsMovingAvgProvider } from 'ui/agg_types/metrics/moving_avg';
-import { AggTypesMetricsSerialDiffProvider } from 'ui/agg_types/metrics/serial_diff';
+import { derivativeMetricAgg } from 'ui/agg_types/metrics/derivative';
+import { cumulativeSumMetricAgg } from 'ui/agg_types/metrics/cumulative_sum';
+import { movingAvgMetricAgg } from 'ui/agg_types/metrics/moving_avg';
+import { serialDiffMetricAgg } from 'ui/agg_types/metrics/serial_diff';
 import { VisProvider } from 'ui/vis';
 import StubbedIndexPattern from 'fixtures/stubbed_logstash_index_pattern';
 
 const metrics = [
-  { name: 'derivative', title: 'Derivative', provider: AggTypesMetricsDerivativeProvider },
-  { name: 'cumulative_sum', title: 'Cumulative Sum', provider: AggTypesMetricsCumulativeSumProvider },
-  { name: 'moving_avg', title: 'Moving Avg', provider: AggTypesMetricsMovingAvgProvider },
-  { name: 'serial_diff', title: 'Serial Diff', provider: AggTypesMetricsSerialDiffProvider },
+  { name: 'derivative', title: 'Derivative', provider: derivativeMetricAgg },
```

Comment:
Maybe we could rename that field from `provider` to something like `agg` since it's not really a provider function anymore, that it might just be confusing that way?